application/features/Audio.py

```
@@ -103,26 +110,35 @@ def handleConnected(self):
                               f'module-null-sink sink_name={sink_name} '
         exit_status, _, stdout, _ = self.audio.exec_command_blocking(load_module_command)
         if exit_status != 0:
-            print(f'AudioWebSocket: audio_id={audio_id}: unable to load pactl module-null-sink sink_name={sink_name}')
+            logger.warning(
+                "AudioWebSocket: audio_id=%s: unable to load pactl module-null-sink sink_name=%s",
+                audio_id,
+                sink_name
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)